File: _includes/SampleCode/SampleCode/Samples/Graph/CustomStories.cpp
#include "pch.h"
using namespace winsdkfb;
using namespace winsdkfb::Graph;
...

// Get active session
FBSession^ sess = FBSession::ActiveSession;
if (sess->LoggedIn)
{
    // Create custom story with action:try and object:scenario
    // This will be published to Facebook as :
    // <user> tried <a scenario> from Sample Application

    // Set parameters for custom story
    PropertySet^ parameters = ref new PropertySet();
    // Set object type parameter
    // Object type: scenario
    String^ customObjectInstance = "{" +
        "\"type\":\"fbsdk_sample_app:scenario\"," + "\"title\":\"Custom Story\"" +
        "}";

    parameters->Insert("scenario", customObjectInstance);

    // Get current user
    FBUser^ user = sess->User;

    // Set Graph api path for custom story (action:try)
    String^ path = user->Id + L"/fbsdk_sample_app:try";

    // Create a json class factory with a class (FBReturnObject class)
    // that can receive and parse the json response returned
    FBJsonClassFactory^ fact = ref new FBJsonClassFactory([](String^ JsonText) ->
        Object^
    {
        auto returnObject = ref new FBReturnObject();
        returnObject->Id = Windows::Data::Json::JsonObject::Parse(JsonText)->GetNamedString("id");
        return returnObject;
    });

    FBSingleValue^ sval = ref new FBSingleValue(graphPath, parameters, fact);
    create_task(sval->PostAsync()).then([this](FBResult^ result)
    {
        if (result->Succeeded)
        {
            FBReturnObject^ response = static_cast<FBReturnObject ^>(result->Object);
        }
        else
        {
            // Posting failed
        }
    });
}

File: _includes/SampleCode/SampleCodeCs/Samples/Graph/CustomStories.cs
using System;
using Windows.Foundation.Collections;
using Newtonsoft.Json;
using winsdkfb;
using winsdkfb.Graph;

namespace SampleCode
{
    public static partial class SampleFunc
    {
        static async void CustomStory()
        {
            // Get active session
            FBSession sess = FBSession.ActiveSession;
            if (sess.LoggedIn)
            {
                // Create custom story with action:try and object:scenario
                // This will be published to Facebook as:
                // <user> tried <a scenario> from Sample Application

                // Set parameters for custom story
                PropertySet parameters = new PropertySet();
                // Set object type parameter
                // Object type: scenario
                string customObjectInstance = "{" +
                    "\"title\":\"Custom Story\"" +
                    "}";
                parameters.Add("scenario", customObjectInstance);
                // Get current user
                FBUser user = sess.User;
                // Set Graph api path for custom story (action:try)
                string path = user.Id + "/fbsdk_sample_app:try";
                var factory = new FBJsonClassFactory(s =>
                {
                    return JsonConvert.DeserializeObject<FBReturnObject>(s);
                });

                var singleValue = new FBSingleValue(path, parameters, factory);
                var result = await singleValue.PostAsync();
                if (result.Succeeded)
                {
                    var response = result.Object as FBReturnObject;
                }
                else
                {
                    // Posting failed
                }
            }
        }
    }
}

